tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -62,6 +63,14 @@ services:
     entrypoint: ["redis-server"]
     command: ["/usr/local/etc/redis/redis.conf"]
 {% endfor %}
+receptor_1:
+  image: quay.io/ansible/awx-ee:devel
```

Comment:
when the other merge happens, we'll want to change this to the main awx image.

This is because you can't really have the code compatible with both. The awx-ee is missing `podman`, which means that you can't run with isolation. The awx image lacks `ansible`, which means that you can't run anything _outside_ of the container.